group4blandine/group4

```diff
@@ -0,0 +1,7 @@
+Sinclair
+William
+Blandine
+Serge
+JC
```

Comment:
I just want to add some commands

curl.ifconfig.me
cat /etc/*rel/ | grep -i JC

Let me know if this commad work for you!

Reply:
it will list everything containing JC

Reply:
any concern?